src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.strategy;
+
+import static core.basesyntax.storage.FruitStorage.fruitStorage;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.storage.FruitTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+    private BalanceHandler balanceHandler;
+    private PurchaseHandler purchaseHandler;
+
+    @BeforeEach
+    void init() {
+        balanceHandler = new BalanceHandler();
```

Comment:
```suggestion
```
Make your testing isolated. If you need to add some data to the storage make it directly